src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/EnvironmentVariableProjectSystemOptions.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.ProjectSystem.Utilities;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS
+{
+    [Export(typeof(IProjectSystemOptions))]
+    internal class EnvironmentVariableProjectSystemOptions : IProjectSystemOptions
+    {
+        private readonly IEnvironmentHelper _environment;
+
+        [ImportingConstructor]
+        public EnvironmentVariableProjectSystemOptions(IEnvironmentHelper environment)
+        {
+            Requires.NotNull(environment, nameof(environment));
+
+            _environment = environment;
+        }
+
+        public bool IsProjectOutputPaneEnabled
+        {
+            get
+            {
+#if DEBUG
+                return true;
+#else
+                return IsEnabled("PROJECTSYSTEM_PROJECTOUTPUTPANEENABLED");
+#endif
+            }
+        }
+
+        private bool IsEnabled(string variable)
+        {
+            string value = _environment.GetEnvironmentVariable(variable);
```

Comment:
I think this is temporarly, and will probably switch this back to registry - and write a script that automatically sets this based on @tannergooding's suggestion.